### PR TITLE
Add newer version support

### PR DESCRIPTION
**Pull request by vishesh92**

### CODE REVIEW

File: tasks/configure_new.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+- name: PGBouncer | Create database conf directory
+  file:
+    path: "{{ pgbouncer_conf_dir }}"
+    state: directory
+    recurse: yes
+
+- name: PGBouncer | Configure database conf
+  template:
+    src: ../templates/database.ini.j2
```
Comment (manjitkumar) [inline tasks/configure_new.yml]: let's keep it in quotes. or may be as a variable?

File: tasks/configure_new.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    dest: "{{ pgbouncer_conf_dir }}/{{ item.name }}.ini"
+  with_items: "{{ pgbouncer_database_aliases }}"
```
Comment (manjitkumar) [inline tasks/configure_new.yml]: in new release of ansible 2.2 it's required to have a default input as empty list if using with items.

File: tasks/configure_new.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  register: pgbouncer_database_conf_list
+  tags:
+    - deploy
+
+- set_fact:
+    pgbouncer_database_conf_list: "{{ pgbouncer_database_conf_list.stdout_lines }}"
+
+- name: PGBouncer | Configure pgbouncer
+  template:
+    src: ../templates/pgbouncer.ini.j2
```
Comment (manjitkumar) [inline tasks/configure_new.yml]: same as above?

File: tasks/configure_new.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    dest: "{{ pgbouncer_conf_file }}"
+  register: pgbouncer_conf_changed
+  tags:
+    - deploy
+
+- name: PGBouncer | Configure auth_users list
+  template:
+    src: ../templates/userlist.txt.j2
```
Comment (manjitkumar) [inline tasks/configure_new.yml]: here as well

File: tasks/main.yml [hunk 5]
```diff
@@ -1,37 +1,23 @@
...
-  replace:
-    regexp: '^START=0'
-    replace: 'START=1'
-    dest: /etc/default/pgbouncer
-  when: pgbouncer_start_on_boot == True
+- include: configure_old.yml
+  when: "not pgbouncer_include_supported"
+
+- include: configure_new.yml
+  when: "pgbouncer_include_supported"
```
Comment (manjitkumar) [inline tasks/main.yml]: Why don't we use it like this "{{ variable }}"?

File: tasks/configure_new.yml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    - deploy
+
+- name: PGBouncer | get conf file list
+  command: ls "{{ pgbouncer_conf_dir }}"
+  register: pgbouncer_database_conf_list
+  tags:
+    - deploy
+
+- set_fact:
+    pgbouncer_database_conf_list: "{{ pgbouncer_database_conf_list.stdout_lines }}"
```
Comment (manjitkumar) [inline tasks/configure_new.yml:29]: if --deploy tag will be given while running playbook this fact will not be set.  you may see unexpected result for var `pgbouncer_database_conf_list`

**Comment by mukgupta** [comment]
Approved

**Pull request merged**